source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -533,7 +535,14 @@ export class Template {
      */
     expand(context: IEvaluationContext): any {
         this._context = new EvaluationContext(context);
-
         return this.internalExpand(this._preparedPayload);
     }
+
+    /**
+     * Getter method for the array of error strings
+     * @returns An array storing any errors that occurred while expanding the template
+     */
+    public getTemplateExpansionErrors(): string[] {
```

Comment:
I really want to see the message is show in designer

Reply:
Created an issue, and will resolve in a separate PR.

https://github.com/microsoft/AdaptiveCards/issues/7442